### PR TITLE
Allow whitelisting immediately 

### LINKED ISSUE #446: Update Register Round Extrinsic

### PR DESCRIPTION
**Pull request by felixfaisal**
This PR resolves #446 
Few things to note: 
- Some of the code is intentionally commented out instead of removing it completely, As I might require it to understand other extrinsic
- The commented out code will be removed as part of the refactoring and testing issue.

### CODE REVIEW

File: pallets/polkadex-ido/src/lib.rs [hunk 1]
```diff
@@ -384,7 +381,7 @@ pub mod pallet {
                         <InfoFundingRound<T>>::remove(&round_id);
                         Self::deposit_event(Event::CleanedupExpiredRound(round_id));
                     }
-                }
+                } */
             }
```
Comment (serhii-temchenko) [inline pallets/polkadex-ido/src/lib.rs]: I assume the whole block can be commented out. There is no need to loop thru here.

**Comment by serhii-temchenko** [comment]
> ...
> * The commented out code will be removed as part of the refactoring and testing issue.

Do we have existed Issue already for this purpose? If so, can we explicitly define issue number in such places to easier navigate when time will come? E.g.:
`// TODO: [Issue #446] This is commented out for now, Will be refactored and removed in the future`

File: pallets/polkadex-ido/src/lib.rs (hunk 1 above)
Comment (felixfaisal) [reply pallets/polkadex-ido/src/lib.rs]: Updated ✅

**Review by serhii-temchenko — APPROVED** [review]

**Comment by felixfaisal** [comment]
Will be changing entire IDO implementation

**Pull request closed without merging**